api/entries.js

```diff
@@ -143,15 +143,14 @@ module.exports = {
     },
 
     getActions : function(req, res){
-        var actions = [];
-
-        var game = req.params.game;
-
-        actions.push("NPC");
-        actions.push("Rat");
-        actions.push("Pickup");
-
-        return res.send(actions);
+        return EntryModel.find().distinct('action', function(err, result){
```

Comment:
Can this be game-specific? I'll provide the string (ex: "Fallout New Vegas"), and get back the actions for that game?